debezium-core/src/test/java/io/debezium/config/AbstractFieldTest.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.config;
+
+import static org.junit.Assert.fail;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.junit.Test;
+
+/**
+ * @author Randall Hauch
+ *
+ */
+public abstract class AbstractFieldTest {
```

Comment:
I added this to test for duplicates. It's an abstract implementation here in core that's used by the different connectors to test for duplicates in the respective connector config (ALL_FIELDS).